src/Extension/Extension.php

```diff
@@ -307,6 +313,25 @@ public function hasAssets()
         return realpath($this->path.'/assets/') !== false;
     }
 
+    public function copyAssetsTo(FilesystemInterface $target)
+    {
+        if (! $this->hasAssets()) {
+            return;
+        }
+
+        $mount = new MountManager([
+            'source' => $source = new Filesystem(new Local($this->getPath().'/assets')),
```

Comment:
Wouldn't it make more sense to have a global Filesystem that gets the extension subdirectory in the code below. That would reduce the need of object instantiation. 50 extensions migrating assets otherwise generate a pretty large memory footprint due to these instantiations..

Reply:
Do we support enabling multiple extensions at once? Because if we don't, the only place that would be affected by this is the installer, which a) isn't particularly performance-critical and b) will soon lose this functionality (see my list of next steps). So I'd take the lazy route here...